test/integration_test.go

```diff
@@ -1221,6 +1221,15 @@ func (ts *IntegrationTestSuite) TestWorkflowWithParallelMutableSideEffects() {
 	ts.NoError(ts.executeWorkflow("test-wf-parallel-mutable-side-effects", ts.workflows.WorkflowWithParallelMutableSideEffects, nil))
 }
 
+func (ts *IntegrationTestSuite) TestWorkflowTypedSearchAttributes() {
```

Comment:
Can we have a test that confirms `TemporalChangeVersion` is set properly and accessible with typed search attributes?

Reply:
:+1: Will add